### ISSUE TITLE
Paintings move by 1 block when you re-enter the world

### ISSUE THREAD

**Issue by vinnyg0621** [Luke100000/ImmersivePaintings#85]
**Describe the bug**
When I re-enter my world, paintings have moved by at least one block up or up-left or up-right corner

**To Reproduce**
1. place a modded painting on a wall
2. quit the world
3. re-join

**Expected behavior**
I didn't expect paintings to move at all.

**Screenshots**
_coming soon!_

**Additional context**
I want to use this mod, but I can't keep using it if paintings keep moving, but I want to use it! so please fix. btw, someone also said it's fixed in 2.0, which probably doesn't exist. I can't find that version anywhere!

**Minecraft version**
MinecraftForge 1.20.1 - 47.3.10

**Comment by Luke100000** [comment]
It should have been fixed in 0.1.7, I assume your someone meant 0.2.0. https://github.com/Luke100000/ImmersivePaintings/blob/1.19.2/changelog.md

Can you reconstruct the behavior with only immersive paintings installed, on every position? Does the orientation matter?

**Comment by vinnyg0621** [comment]
yeah i just tried just immersive paintings and dark paintings installed on my test world. the paintings didn't move when I re-entered it
